Mathlib/LinearAlgebra/AffineSpace/Basis.lean

```diff
@@ -277,6 +275,24 @@ instance instAddAction : AddAction V (AffineBasis ι k P) :=
 
 end Ring
 
+section CommRing
+variable [CommRing k] [Module k V] (b : AffineBasis ι k P) {s : Finset ι} {i j : ι} (e : ι ≃ ι')
+
+instance instSMul : SMul kˣ (AffineBasis ι k V) where
```

Comment:
Does this generalize to any group action on `k`?

Reply:
Yes